### PR TITLE
LRPS-91834 Segments Administration

### PR DESCRIPTION
**Pull request by stephenkostas**
Added intro draft, and created exercises based on examples provided by product team.
https://issues.liferay.com/browse/LPS-91384

### CODE REVIEW

File: discover/portal/articles/XX-segmentation-and-personalization/01-segmentation-intro.markdown [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Segmentation and Personalization
+
+Segmentation and Personalization provides a way to evaluate users so that you 
+can dynamically provide personal experiences that are relevant to each user.
```
[the remark this answers is not in the document]
Comment (stephenkostas) [reply discover/portal/articles/XX-segmentation-and-personalization/01-segmentation-intro.markdown]: Hey, so we should definitely have a call to discuss the specifics of the intro and how I should talk about the various aspects of it. This was created as somewhat of a placeholder. The general goal in our section intros is to do a high level introduction of the topic as a whole, address each component separately and then provide step by step instructions of a basic use case.

What's the best term to use for describing "users"? Should I say "visitors"?

File: discover/portal/articles/XX-segmentation-and-personalization/02-segment-editor.markdown [hunk 2]
```diff
@@ -42,3 +42,7 @@ information. Things like the current browser, screen resolution, and platform.
 You can use this to target categories like "iPhone users" or "Internet Explorer 
 Users."
```
[the remark this answers is not in the document]
Comment (stephenkostas) [reply discover/portal/articles/XX-segmentation-and-personalization/02-segment-editor.markdown]: I did have that under Session Properties, unless there was an issue with how the doc is displaying on github...

[the remark this answers is not in the document]
Comment (AngeloYoun) [reply discover/portal/articles/XX-segmentation-and-personalization/02-segment-editor.markdown]: Oh yah, it was just github haha. The other thing would be url or cookies would be much more useful examples. For example creating an email campaign with a promo code in url parameter, then segmenting people with that param in their url or targeting people with a cookie that was sent when they downloaded a pdf. Or even just logged in vs logged out.

File: discover/portal/articles/XX-segmentation-and-personalization/01-segmentation-intro.markdown (hunk 1 above)
[the remark this answers is not in the document]
Comment (AngeloYoun) [reply discover/portal/articles/XX-segmentation-and-personalization/01-segmentation-intro.markdown]: I think visitors is a better term. When referring to a specific subset, i would say authenticated vs unauthenticated to be clear.

**Pull request closed without merging**